### PR TITLE
feat: note card color

### PR DESCRIPTION
**Pull request by devsurajrai**
added the feature to add colors to the notes card. Please review the code.
LIVE: https://deploy-preview-8--graceful-rugelach-517974.netlify.app/

### CODE REVIEW

File: src/components/CreateNoteCard/CreateNoteCard.jsx [hunk 1]
```diff
@@ -3,15 +3,21 @@ import "./create-note-card.css";
 import { useAuthContext } from "../../contexts/authContext";
 import { addNote, updateNote } from "../../util/util";
 import { useNoteContext } from "../../contexts/noteContext";
+import { useColorContext } from "../../contexts/colorContext";
```
Comment (shraddha15-creator) [inline src/components/CreateNoteCard/CreateNoteCard.jsx:6]: You can use index.js file in your context folder and then import all contexts from there

File: src/components/SideBar/SideBar.jsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { useAuthContext } from "../../contexts/authContext";
+import { useColorContext } from "../../contexts/colorContext";
```
Comment (shraddha15-creator) [inline src/components/SideBar/SideBar.jsx:2]: same for this

File: src/index.js [hunk 3]
```diff
@@ -9,6 +9,8 @@ import { ShowNoteContextProvider } from "./contexts/showNotesContext";
 import { SignUpLoginValidateContextProvider } from "./contexts/signUpLoginValidationContext";
 import { AuthContextProvider } from "../src/contexts/authContext";
 import { NoteContextProvider } from "./contexts/noteContext";
+import { ColorContextProvider } from "./contexts/colorContext";
```
Comment (shraddha15-creator) [inline src/index.js:12]: here as well

**Review by shraddha15-creator — APPROVED** [review]
Added some suggestions
else code looks good and clean
UI color theme is nice.

File: src/components/CreateNoteCard/CreateNoteCard.jsx (hunk 1 above)
Comment (devsurajrai) [reply src/components/CreateNoteCard/CreateNoteCard.jsx:6]: thanks will implement that

Comment (Dev-AkshatGupta) [reply src/components/CreateNoteCard/CreateNoteCard.jsx:6]: Not only that but try using absolute imports there to will help you alot

File: src/components/CreateNoteCard/CreateNoteCard.jsx [hunk 4]
```diff
@@ -83,7 +104,6 @@ export const CreateNoteCard = ({
                           userToken,
                           authDispatch,
                           setNote,
-                          auth.notes,
                           setIsEditing
                         )
                       }
```
Comment (Dev-AkshatGupta) [inline src/components/CreateNoteCard/CreateNoteCard.jsx:109]: Please use something for the indentation

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems good to me just look at the suggestions given
You can merge it

**Comment by sushmita12345** [comment]
- In live preview, Get Started, Login, Create New Note, Edit Note all button disappears by hovering on it, so fix it.
- Card color functionality working fine.
- Go with the above suggested changes, you can proceed to merge.

**Review by sushmita12345 — APPROVED** [review]

**Pull request merged**